fem/integ/bilininteg_convection_pa.cpp

```diff
@@ -450,8 +450,8 @@ void PAConvectionApply3D(const int ne,
    const int NE = ne;
    const int D1D = T_D1D ? T_D1D : d1d;
    const int Q1D = T_Q1D ? T_Q1D : q1d;
-   MFEM_VERIFY(D1D <= MAX_D1D, "");
-   MFEM_VERIFY(Q1D <= MAX_Q1D, "");
+   MFEM_VERIFY(D1D <= DeviceDofQuadLimits::Get().MAX_D1D, "");
+   MFEM_VERIFY(D1D <= DeviceDofQuadLimits::Get().MAX_Q1D, "");
```

Comment:
Same here. I'll stop adding this comment -- just check for other places where this change was made.